index.php

```diff
@@ -20,6 +20,7 @@
 
 $url = 'https://ddb.glidernet.org/';
 $sender = 'contact@glidernet.org';
+const expirationdelta = 365*24*60*60;
```

Comment:
Maybe a short comment to explain that this is the device registration time in second?

Reply:
And maybe we should add abit more than 1year so this can be done during ARC renewal.
Something like 1y + 3m ?